stable/fluent-bit/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: fluent-bit
-version: 0.2.13
+version: 0.2.14
```

Comment:
This adds features, please bump the minor version.

Reply:
Changed version to 0.2.13.1. Is that inline with what you're suggesting?

Reply:
No, I was suggesting bumping it to 0.3.0.

Reply:
Bumped. Thanks for clarifying.